packages/gallery/src/components/gallery/proGallery/galleryContainer.js

```diff
@@ -701,8 +701,8 @@ export class GalleryContainer extends React.Component {
     this.currentSlideshowViewIdx = idx;
   }
 
-  simulateScrollToItem(item) {
-    item?.offset && this.onGalleryScroll(item.offset);
+  simulateHorizontalScrollToItem(item) {
+    item?.offset && this.onGalleryScroll({left: item.offset.left});
```

Comment:
I don't understand what changed in this PR (other than rename of the method).
Before you change:
`this.onGalleryScroll` was called with `item.offSet` **(it's an object that contains `left`)**

After your change:
this.`this.onGalleryScroll` is called with an object that looks exactly the same ({`left: item.offsetLeft} **(it's an object that contains `left`)**

Unless I'm missing something, the PR just renames the function

Reply:
in this case i want that top won’t change in this.onGalleryScroll, i want to change only the left property. when i send all the obj the top in every scroll become 0, and this affects the scroll position.

Reply:
Understood. Are you sure it won't break anything related to vertical scroll? Because you disabled the vertical scroll behaviour by doing this